<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DEMControl</class>
 <widget class="QWidget" name="DEMControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="cellLayout">
     <item row="0" column="2">
      <widget class="QSpinBox" name="cellNbrYSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="cellNbrXSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="cellNbrTextLabel">
       <property name="text">
        <string>Cells number:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="cellSizeXSpinBox">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="cellSizeYSpinBox">
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="cellSizeTextLabel">
       <property name="text">
        <string>Cells size:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="heightFilterTextLabel">
       <property name="text">
        <string>Height filter:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="heightFilterMinSpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>-2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="heightFilterMaxSpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="transformationLayout">
     <item>
      <widget class="QLabel" name="translationTextLabel">
       <property name="text">
        <string>Translation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="xSpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ySpinBox">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lineSizeLayout">
     <item>
      <widget class="QLabel" name="lineSizeTextLabel">
       <property name="text">
        <string>Line size:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lineSizeSpinBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="smoothLinesCheckBox">
       <property name="text">
        <string>Smooth</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ColorChooser" name="colorChooser" native="true"/>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="showLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="showDEMCheckBox">
       <property name="text">
        <string>Show DEM</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <action name="testAction">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorChooser</class>
   <extends>QWidget</extends>
   <header>ColorChooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineSizeSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DEMControl</receiver>
   <slot>lineSizeChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>smoothLinesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DEMControl</receiver>
   <slot>smoothLinesToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showDEMCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DEMControl</receiver>
   <slot>showDEMToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellNbrXSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellNbrYSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellSizeXSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cellSizeYSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightFilterMinSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightFilterMaxSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>DEMControl</receiver>
   <slot>demChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lineSizeChanged(double)</slot>
  <slot>smoothLinesToggled(bool)</slot>
  <slot>showDEMToggled(bool)</slot>
  <slot>demChanged()</slot>
 </slots>
</ui>
